doc/stdenv/stdenv.chapter.md

```diff
@@ -309,13 +309,51 @@ The attribute can also contain a list, a script followed by arguments to be pass
 passthru.updateScript = [ ../../update.sh pname "--requested-release=unstable" ];
 ```
 
-The script will be run with the `UPDATE_NIX_NAME`, `UPDATE_NIX_PNAME`, `UPDATE_NIX_OLD_VERSION` and `UPDATE_NIX_ATTR_PATH` environment variables set respectively to the name, pname, old version and attribute path of the package it is supposed to update.
+Finally, the attribute can be an attribute set, which, in addition to the command, can optionally provide the canonical attribute path to use for the list of extra features the script supports.
+
+```nix
+passthru.updateScript = {
+  command = [ ../../update.sh pname ];
+  attrPath = pname;
+  supportedFeatures = [ … ];
+};
+```
+
+##### How update scripts are executed? {#var-passthru-updateScript-execution}
+
+For information about how to run update scripts, run `nix-shell maintainers/scripts/update.nix` in the root of Nixpkgs repository. The script offers several modes for finding and executing update scripts.
+
+Each script will be run with the `UPDATE_NIX_NAME`, `UPDATE_NIX_PNAME`, `UPDATE_NIX_OLD_VERSION` and `UPDATE_NIX_ATTR_PATH` environment variables set respectively to the name, pname, old version and attribute path of the package it is supposed to update.
 
 ::: {.note}
 The script will be usually run from the root of the Nixpkgs repository but you should not rely on that. Also note that the update scripts will be run in parallel by default; you should avoid running `git commit` or any other commands that cannot handle that.
 :::
 
-For information about how to run the updates, execute `nix-shell maintainers/scripts/update.nix`.
+`maintainers/scripts/update.nix` also supports automatically creating commits when running it with `--argstr commit true`. Update scripts can opt to override the commit contents by adding `commit` to `supportedFeatures` and [modifying the script accordingly](#var-passthru-updateScript-commit). It might be useful, for example, for including news link in the commit message.
+
+##### Supported features {#var-passthru-updateScript-supported-features}
+###### `commit` {#var-passthru-updateScript-commit}
```

Comment:
Maybe we should rename the feature to something like `commitProtocol`, as it might be confusing (e.g. https://github.com/NixOS/nixpkgs/pull/193248#discussion_r981779706).

Reply:
This explanation could be more verbose. I had difficulty in understanding the idea initially. Maybe the explanation found in the linked comment can be added to this PR.

Reply:
@romildo I have added a introductory sentence:

> This feature allows update scripts to *ask* `update.nix` to create Git commits.

And mentioned again in the tip that update scripts should not actually create commits.